examples/transforms/align.rs

```diff
@@ -147,37 +141,33 @@ fn draw_random_axes(mut gizmos: Gizmos, query: Query<&RandomAxes>) {
 }
 
 // Actually update the ship's transform according to its initial source and target
-fn rotate_ship(mut ship: Query<(&mut Ship, &mut Transform)>) {
+fn rotate_ship(mut ship: Query<(&mut Ship, &mut Transform)>, time: Res<Time>) {
     let (mut ship, mut ship_transform) = ship.single_mut();
 
     if !ship.in_motion {
         return;
     }
 
-    let start = ship.initial_transform.rotation;
     let end = ship.target_transform.rotation;
 
-    let p: f32 = ship.progress.into();
-    let t = p / 100.;
-
-    *ship_transform = Transform::from_rotation(start.slerp(end, t));
+    ship_transform
+        .rotation
+        .smooth_nudge(&end, 3.0, time.delta_seconds());
 
-    if ship.progress == 100 {
+    if ship_transform.rotation.angle_between(end) <= f32::EPSILON {
```

Comment:
Is this comparison always enough? f32::eps is pretty small.

Reply:
The `in_motion` field this alters doesn't actually care much about whether this gets changed here, so maybe the best thing would just be to delete this entire conditional anyway. (That would be a bit weird conceptually though.)